tiledb/sm/tile/tile_metadata_generator.cc

```diff
@@ -501,6 +501,7 @@ void TileMetadataGenerator::process_tile(
         process_tile<int64_t>(tile, tile_validity);
         break;
       case Datatype::STRING_ASCII:
+      case Datatype::CHAR:
```

Comment:
Could this be missing from here as well  :
https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/query/result_tile.cc#L1192 ?

Reply:
Looks that way. I filed https://app.shortcut.com/tiledb-inc/story/15528/resulttile-set-compute-results-func-should-have-a-case-for-datatype-char so we can get this one in faster in 2.7.

Reply:
ok, thanks. How about Datatype::BLOB? Do we need a case here for that too?

Reply:
BLOB has no min/max or sum metadata. For ResultTile::set_compute_results_func, I don't think we allow BLOB as a dimension so we should be ok too.

Reply:
@ypatia actually, you are right, we do need a case for it as we still want the null count... I will fix this PR. Good catch!